moj_analytics/auth0_client.py

```diff
@@ -138,6 +142,10 @@ def get_all(self, resource_class):
 
         resources = self.request('GET', endpoint)
 
+        if not resources:
+            log.msg("Error getting resource.", response=response)
+            raise GetResourceError(response)
+
```

Comment:
My memory of this is very rusty but are we sure this is what we want?

Isn't `get_all` used to check if a webapp resources' are there or not as well? No resources doesn't necessary mean something went wrong.

Wouldn't this raise an exception for a new webapp when the resource checks if its resources are there before creating them?

I may be wrong and completely missing the point or not remember correctly, I only had a quick look at the diff but not the code, but something worth checking probably.

Reply:
Ugh... I think you may be right (my memory is also very rusty). Let me check again.